modules/virt-storage-wizard-fields-web.adoc

```diff
@@ -48,34 +48,42 @@
 |Storage Class
 |
 |The storage class that is used to create the disk.
-
-a|Advanced -> Volume Mode
-[NOTE]
-====
-Default values are used from the storage profile.
-====
-|
-|Defines whether the persistent volume uses a formatted file system or raw block state. Default is *Filesystem*.
 |===
 
 [id="virt-storage-wizard-fields-advanced-web_{context}"]
 [discrete]
 == Advanced storage settings
 
-[cols="5a,3a,5a"]
-|===
-|Name | Parameter |  Description
+The following advanced storage settings are available for *Blank*, *Import via URL*, and *Clone existing PVC* disks. These parameters are optional. If you do not specify these parameters, the system uses the default values from the `kubevirt-storage-class-defaults` config map.
```

Comment:
the `kubevirt-storage-class-defaults` config map is removed from CNV 4.11 and 4.12.

Reply:
@bgaydosrh FYI

Reply:
Thanks @gouyang - Is it safe to say that if the parms aren't specified that it uses the default values from the Storage Profile? Or would you prefer I just drop the last sentence referring to the map entirely? 
Bob

Reply:
Thanks @gouyang - noted.